grai-server/app/entrypoint.sh

```diff
@@ -24,5 +24,12 @@ python manage.py collectstatic --no-input
 export BEGIN_LOGGING="True"
 python manage.py shell < "init.py"
 
+# Seed database from json files
```

Comment:
Fixtures need to be called with `loaddata` and a fixture name.  This little script calls all the files in `/seeds` alternatively I can move the json files under a fixtures folder in a module, however then the fixtures need to be called by name in `entrypoint.sh`.

Reply:
See https://github.com/grai-io/grai-core/pull/67 for how this would work

Reply:
Ahh, got it. I misread the fixtures docs and thought it would only load once if placed explicitly under fixtures.

What's weird about loading fixtures in the init script is that they will be loaded every time the container starts overwriting whatever currently exists in the database which just feels sort of icky. What about including the initially populated data as part of a migration instead of in the fixture?